core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/upgrade.yml

```diff
@@ -116,6 +140,24 @@
     # Process the same configuration files as for 'apply'
     - name: PostgreSQL | Copy configuration files
       block:
+        - debug:
+            var: _specification.extensions
+          vars:
+            _files:
+              - pg_hba.conf
+              - postgresql-epiphany.conf
+              - postgresql.conf
+            _pg_data_dir: "{{ new_version.pg.data_dir[ansible_os_family] }}" # used in postgresql.conf
+            _extensions:
+              extensions:
+                pgaudit:
+                  enabled: "{{ pg_is_pgaudit_used }}"
+                replication:
+                  enabled: "{{ pg_is_repmgr_used }}"
+            # Values from manifest are used when it's not possible to do changes after 'apply'
+            # such as enabling/disabling extensions
+            _specification: "{{ upgrade_vars.specification | combine(_extensions, recursive=True) }}"
+
```

Comment:
Duplication?

Reply:
Thanks! It was debug code left by accident.